website/blog/2025-03-10-7.27.0.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title:  "7.27.0 Released: discard bindings proposal, and better ecosystem alignment"
+authors: team
+date:   2025-03-10 0:00:00
+categories: announcements
+share_text: "Babel 7.27.0 Released"
+---
+
+Babel 7.27.0 is out!
+
+Babel now supports parsing and transforming the Stage 2 [discard binding proposal](https://github.com/tc39/proposal-discard-binding), for cases were you are forced to define an unused variable. Babel's code printer will now also print the correct [import attributes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import/with) syntax by default, rather than expecting its users to explicitly enable an option for it.
```

Comment:
`were` -> `where` typo?

```suggestion
Babel now supports parsing and transforming the Stage 2 [discard binding proposal](https://github.com/tc39/proposal-discard-binding), for cases where you are forced to define an unused variable. Babel's code printer will now also print the correct [import attributes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import/with) syntax by default, rather than expecting its users to explicitly enable an option for it.
```